services/vision/client_test.go

```diff
@@ -69,7 +69,7 @@ func TestClient(t *testing.T) {
 		client, err := vision.NewClientFromConn(context.Background(), conn, "", vision.Named(testVisionServiceName), logger)
 		test.That(t, err, test.ShouldBeNil)
 
-		dets, err := client.Detections(context.Background(), &image.RGBA{}, nil)
+		dets, err := client.Detections(context.Background(), image.NewRGBA(image.Rect(0, 0, 10, 20)), nil)
```

Comment:
I'm assuming this was changed to avoid passing in an empty image and getting an error?

Reply:
yes exactly - it did not like having to encode an empty struct